core-services/image-mirroring/openshift/mapping_origin_4_13

```diff
@@ -221,6 +221,7 @@ registry.ci.openshift.org/origin/4.13:must-gather quay.io/openshift/origin-must-
 registry.ci.openshift.org/origin/4.13:network-interface-bond-cni quay.io/openshift/origin-network-interface-bond-cni:4.13 quay.io/openshift/origin-network-interface-bond-cni:4.13.0
 registry.ci.openshift.org/origin/4.13:network-metrics-daemon quay.io/openshift/origin-network-metrics-daemon:4.13 quay.io/openshift/origin-network-metrics-daemon:4.13.0
 registry.ci.openshift.org/origin/4.13:network-tools quay.io/openshift/origin-network-tools:4.13 quay.io/openshift/origin-network-tools:4.13.0
+registry.ci.openshift.org/origin/4.13:nmstate-console-plugin quay.io/openshift/origin-nmstate-console-plugin:4.13 quay.io/openshift/origin-nmstate-console-plugin:4.13.0
```

Comment:
there are no other nmstate specific images today?  are any more anticipated?

Reply:
According to the kubernetes-nmstate CSV:
* [nmstate-handler](https://github.com/openshift/kubernetes-nmstate/blob/master/manifests/stable/manifests/kubernetes-nmstate-operator.clusterserviceversion.yaml#L189)
* [nmstate-operator](https://github.com/openshift/kubernetes-nmstate/blob/master/manifests/stable/manifests/kubernetes-nmstate-operator.clusterserviceversion.yaml#L194)

These images are hosted at `quay.io/openshift/origin-*` as well, however I don't see their image-mirroring configuration in this repo. This may have been configured somewhere else.

Reply:
those aren't real values, they get substituted by the actual image SHA that gets built, via this mapping:

https://github.com/openshift/kubernetes-nmstate/blob/master/manifests/stable/manifests/image-references

Reply:
so maybe i need to ask a different question:  what is the motivation to mirror this particular image?  we generally only do this when there is a direct public consumption need

Reply:
see:
https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift/GUIDELINES.md#which-images-to-mirror

Reply:
and also https://docs.ci.openshift.org/docs/how-tos/mirroring-to-quay/

Reply:
ok, in case of `nmstate-operator` and `nmstate-handler` images, their code is located at the same repo (https://github.com/openshift/kubernetes-nmstate), thus can be built by the ci-operator directly:

```
$ wget https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_kubernetes-nmstate/357/pull-ci-openshift-kubernetes-nmstate-master-e2e-handler-ovn-ipv4/1638405886600810496/artifacts/e2e-handler-ovn-ipv4/gather-extra/artifacts/pods.json

$ jq -r '.items[] | select(.metadata.name=="nmstate-operator-7465c66478-rbp8j").spec.containers[0].image' 
pods.json 
registry.build05.ci.openshift.org/ci-op-1izgt0qk/pipeline@sha256:2f52a41fa609c96aa922b8731e2904e257125e573a0f9d1e8f6988c4287137e1

$ jq -r '.items[] | select(.metadata.name=="nmstate-handler-9bq7k").spec.containers[0].image' pods.json 
registry.build05.ci.openshift.org/ci-op-1izgt0qk/pipeline@sha256:cc0b193abc8e4acdff39352dbed76731c1cd08ee1a7ddc72c7364407fc7c1403
```

in case of `nmstate-console-plugin` pod, its image code is in another repo: https://github.com/openshift/nmstate-console-plugin
Could you please suggest how can I reference a non-upstream image pullspec for `nmstate-console-plugin`, as required by [this comment](https://github.com/openshift/kubernetes-nmstate/pull/357#discussion_r1139265486)?

Thanks